### DISCUSSION TITLE
sorting by dateAscending help

### DISCUSSION THREAD

**Discussion by billpetro** [jamiebrynes7/obsidian-todoist-plugin#145] [Q&A]
This plugin works great, though I'd like to sort by dateAscending order. This syntax throws an error: "Unexpected string in JSON at position 40"

```todoist
{
"name": "My Tasks",
"filter": "today"
"sorting": "dateAscending"
}
```

**Comment by ignosticism** [comment]
This works for me:

````todoist
{
"name": "My Tasks",
"filter": "today | overdue",
"sorting": ["dateAscending"]
}
````

**Comment by billpetro** [comment]
Perfect, once I get all the "commas" in the right places.

Thanks, this does it!




[image: photo]
*Bill Petro*
m:719.963.0229 | ***@***.*** | w:billpetro.com
<http://linkedin.com/in/billpetro>  <http://twitter.com/billpetro>
<http://plus.google.com/+BillPetro>  <http://pinterest.com/billpetro>
<http://klout.com/#/billpetro>  <http://ello.co/billpetro>
<http://about.me/billpetro>  <http://facebook.com/HistoryOfTheHolidays>


<http://about.me/billpetro?promo=email_sig>
